app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :comment
+  has_many :post
+  has_many :like
+end
```

Comment:
- [ ] As requested by the project requirements, the user model should have a method that returns the 3 most recent posts. Kindly add that.